hypoddpy/hypodd_relocator.py

```diff
@@ -423,7 +423,7 @@ def _read_event_information(self):
         self.log("Reading all events...")
         catalog = Catalog()
         for event in self.event_files:
-            catalog += readEvents(event)
+            catalog += read_events(event).events[0]
```

Comment:
Can you please revert this change? The effect of your change is that it only take the first event within every file but it very feasible to have multiple events per file.